src/video/android/SDL_androidvideo.c

```diff
@@ -180,7 +180,8 @@ int Android_VideoInit(_THIS)
     mode.format = Android_ScreenFormat;
     mode.w = Android_DeviceWidth;
     mode.h = Android_DeviceHeight;
-    mode.refresh_rate = Android_ScreenRate;
+    mode.refresh_rate.numerator = Android_ScreenRate;
```

Comment:
Android_ScreenRate type should be changed to float, and we need a function to convert a float value to numerator and denominator.

Reply:
```
 64 static float Android_ScreenRate = 0;
...
83     mode.refresh_rate.numerator = (int) (Android_ScreenRate * 1000.0);
184     mode.refresh_rate.denominator = 1000;

```